src/box/errcode.h

```diff
@@ -43,6 +43,13 @@ struct errcode_record {
 	const char *errdesc;
 };
 
+#ifdef TEST_BUILD
+#define ERROR_CODES_TEST(_) \
+	_(ER_TEST_FIRST, 10000,			"Test error first")
```

Comment:
Done.

```diff
diff --git a/src/box/errcode.h b/src/box/errcode.h
index 19a3384ec..8aa821506 100644
--- a/src/box/errcode.h
+++ b/src/box/errcode.h
@@ -44,6 +44,11 @@ struct errcode_record {
 };

 #ifdef TEST_BUILD
+/**
+ * Test error codes are used only for testing. Let's name them with ER_TEST_
+ * prefix (we rely on that in tests). Test error code should be >= 10000
+ * to avoid clash with actual error codes.
+ */
 #define TEST_ERROR_CODES(_) \
        _(ER_TEST_FIRST, 10000,                 "Test error first")
 #else
```